### PR TITLE
v5.0 docs Part 3

### PR DESCRIPTION
**Pull request by jasonblais**
This should complete docs from my end.

### CODE REVIEW

File: source/administration/command-line-tools.rst [hunk 1]
```diff
@@ -435,14 +435,33 @@ mattermost import slack
...
+----------------
 
   Description
     Commands to configure and synchronize AD/LDAP.
 
   Child Command
+    -  `mattermost ldap idmigrate`_ - Migrate the LDAP Id Attribute to a new value
     -  `mattermost ldap sync`_ - Synchronize now
 
+mattermost ldap sync
```
Comment (jasonblais) [inline source/administration/command-line-tools.rst]: @crspeller Is there any more concrete steps or information we can give to admins when doing the migration?

File: source/administration/config-settings.rst [hunk 2]
```diff
@@ -3376,6 +3387,31 @@ Specify the color of the SAML login button text for white labeling purposes. Use
 
 ________
 
+Cluster Settings
```
Comment (jasonblais) [inline source/administration/config-settings.rst]: @lieut-data Can you help review the accuracy of these new cluster settings

File: source/deployment/sso-ldap.md [hunk 3]
```diff
@@ -125,6 +125,14 @@ If the issue persists, try performing a sync with the **User Filter** field blan
 
 Make sure that you also have at least 1 LDAP user in Mattermost or the sync will not complete.
 
+#### What is the difference betweeen the Username Attribute, ID Attribute and Login ID Attribute?
```
Comment (jasonblais) [inline source/deployment/sso-ldap.md]: @crspeller Can you help review accuracy of the new changes in our LDAP docs

File: source/deployment/sso-saml-ldapsync.rst [hunk 4]
```diff
@@ -29,3 +29,7 @@ Once the synchronization with AD/LDAP is enabled, user attributes are synchroniz
     
     If a user is deactivated from SAML, their session won't expire until they're deactivated from AD/LDAP. However, they won't be able to log back in to Mattermost.
  
+   .. note::
```
Comment (jasonblais) [inline source/deployment/sso-saml-ldapsync.rst]: @crspeller Can you help review accuracy of the new changes in our SAML-LDAP sync docs

File: source/administration/config-settings.rst [hunk 5]
```diff
@@ -3376,6 +3387,34 @@ Specify the color of the SAML login button text for white labeling purposes. Use
...
++----------------------------------------------------------------------------------------------------------------------------------------------------------------------+
+| This feature's ``config.json`` setting is ``"MaxIdleConns": "100"`` with whole number input.                                                                         |
++----------------------------------------------------------------------------------------------------------------------------------------------------------------------+
+
+Maximum Idle Connections per Host
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+Maximum number of idle connections held open to the cluster per hostname.
+
++----------------------------------------------------------------------------------------------------------------------------------------------------------------------+
+| This feature's ``config.json`` setting is ``"MaxIdleConnsPerHost": "120"`` with whole number input.                                                                  |
```
Comment (lieut-data) [inline source/administration/config-settings.rst]: Default value here is `128`, not `120`.

File: source/administration/config-settings.rst [hunk 6]
```diff
@@ -3376,6 +3387,34 @@ Specify the color of the SAML login button text for white labeling purposes. Use
 
 ________
 
+Cluster Settings
+~~~~~~~~~~~~~~~~~~~~~~~~~
+Maximum Idle Connections
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+Maximum number of idle connections held open to the cluster.
```
Comment (lieut-data) [inline source/administration/config-settings.rst]: I might suggest rewording as:
> The maximum number of idle connections held open from one server to all others in the cluster.

File: source/administration/config-settings.rst [hunk 7]
```diff
@@ -3376,6 +3387,34 @@ Specify the color of the SAML login button text for white labeling purposes. Use
...
+
++----------------------------------------------------------------------------------------------------------------------------------------------------------------------+
+| This feature's ``config.json`` setting is ``"MaxIdleConns": "100"`` with whole number input.                                                                         |
++----------------------------------------------------------------------------------------------------------------------------------------------------------------------+
+
+Maximum Idle Connections per Host
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+Maximum number of idle connections held open to the cluster per hostname.
```
Comment (lieut-data) [inline source/administration/config-settings.rst]: I might suggest rewording as:
> The maximum number of idle connections held open from one server to another server in the cluster.

File: source/administration/config-settings.rst [hunk 8]
```diff
@@ -3376,6 +3387,34 @@ Specify the color of the SAML login button text for white labeling purposes. Use
...
+
++----------------------------------------------------------------------------------------------------------------------------------------------------------------------+
+| This feature's ``config.json`` setting is ``"MaxIdleConnsPerHost": "120"`` with whole number input.                                                                  |
++----------------------------------------------------------------------------------------------------------------------------------------------------------------------+
+
+Idle Connection Timeout (in Milliseconds)
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+The number of milliseconds to wait for a response from the cluster after opening a connection.
```
Comment (lieut-data) [inline source/administration/config-settings.rst]: I'd suggest:
> The number of milliseconds to leave an idle connection open between servers in the cluster.

File: source/administration/command-line-tools.rst (hunk 1 above)
Comment (crspeller) [reply source/administration/command-line-tools.rst]: I would be less confusing. We know what AuthData is, but no user is ever supposed to see that. 

I would write: "Run this utility to change the value of your ID Attribute without your users losing their accounts. After running the command you can change the ID Attribute to the new value in your config.json. For example if your current ID Attribute was sAMAccountName and you wanted to change it to objectGUID you would:
1. Wait for an off time when your users won't mind a server restart.
2. Run the command `mattermost ldap idmigrate objectGUID`
3. Edit your config.json and change your IdAttribute field to your new IdAttribute `objectGUID`
4. Restart the Mattermost server"

File: source/deployment/sso-saml-ldapsync.rst (hunk 4 above)
Comment (crspeller) [reply source/deployment/sso-saml-ldapsync.rst]: Yup sounds good.

File: source/administration/command-line-tools.rst (hunk 1 above)
Comment (jasonblais) [reply source/administration/command-line-tools.rst]: Thanks! @crspeller Updated
